pages/index.tsx

```diff
@@ -66,7 +66,7 @@ export default function Home() {
       <div className="p-8 sm:p-28">
         <div className="font-bold text-lg">Features</div>
         <div className="max-w-screen-sm pb-16 text-black text-4xl font-bold">
-          <h2>Making people and actors of the industry the owner of their data.</h2>
+          <h2>Making individuals and industries the sole owners of their data.</h2>
```

Comment:
- I like "people"
- industries is not really an entity which can have ownership